app/views/spree/admin/orders/customer_details/edit.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render 'spree/admin/shared/order_tabs', current: :customer_details %>
+<%= render 'spree/admin/shared/order_tabs', current: :customer %>
```

Comment:
I've changed this in order not to introduce additional variable.

Essentially I'd like to perform the following check
`"#{'active' if current == :customer} nav-link"` referring to tab name (passed as the second positional argument to the `link_to_with_icon`.